src/helpers.js

```diff
@@ -17,6 +17,11 @@ Marionette.isNodeAttached = function(el) {
   return Backbone.$.contains(document.documentElement, el);
 };
 
+// Merge `keys` from `options` onto `this`
+Marionette.mergeOptions = function(options, keys) {
+  if (!options) { return; }
+  _.extend(this, _.pick(options, keys));
```

Comment:
It's a more expressive API. Writing `mergeOptions` vs these two lines in every constructor makes a big difference, I think.